libs/application/template-api-modules/src/lib/modules/templates/financial-statement-individual-election/mappers/helpers.spec.ts

```diff
@@ -0,0 +1,174 @@
+import { ContactType } from '@island.is/clients/financial-statements-inao'
+import * as financialHelper from './helpers'
+import { FinancialElectionIncomeLimit } from './helpers'
+
+describe('Financial statement individual election helpers', () => {
+  describe('getIndividualElectionValues', () => {
+    it.each([
+      { income: FinancialElectionIncomeLimit.LESS, valueStatement: true },
+      { income: FinancialElectionIncomeLimit.GREATER, valueStatement: false },
+    ])('should map correctly', (incomeValueStatementCombo) => {
+      const expectedResult = {
+        noValueStatement: incomeValueStatementCombo.valueStatement,
+        incomeLimit: incomeValueStatementCombo.income,
+        electionId: 'blegh',
+        clientName: 'Fullname Fullname',
+        clientPhone: '1234567',
+        clientEmail: 'email@mockemail.com',
+      }
+
+      const answers = {
+        'election.incomeLimit': expectedResult.incomeLimit,
+        'election.selectElection': expectedResult.electionId,
+        'about.fullName': expectedResult.clientName,
+        'about.phoneNumber': expectedResult.clientPhone,
+        'about.email': expectedResult.clientEmail,
+      }
+
+      const result = financialHelper.getIndividualElectionValues(answers)
+
+      expect(result).toEqual(expectedResult)
+    })
+  })
+
+  describe('getShouldGetFileName', () => {
+    it.each([
+      { input: FinancialElectionIncomeLimit.LESS, expectedOutput: false },
+      { input: FinancialElectionIncomeLimit.GREATER, expectedOutput: true },
+    ])('should map correctly', (inputOutputCombo) => {
+      const answers = {
+        'election.incomeLimit': inputOutputCombo.input,
+      }
+
+      const result = financialHelper.getShouldGetFileName(answers)
+
+      expect(result).toEqual(inputOutputCombo.expectedOutput)
+    })
+  })
+
+  describe('getActorContact', () => {
+    it.each([true, false])('should map correctly', (shouldBeDefined) => {
+      const clientName = 'Fullname Fullname'
+      const nationalId = '1234564321'
+
+      const expectedResult = shouldBeDefined
+        ? {
+            nationalId: nationalId,
+            name: clientName,
+            contactType: ContactType.Actor,
+          }
+        : undefined
+
+      const actor = shouldBeDefined
+        ? { nationalId, scope: ['not', 'applicable?'] }
+        : undefined
+
+      const result = financialHelper.getActorContact(actor, clientName)
+
+      expect(result).toEqual(expectedResult)
+    })
+  })
+
+  describe('getInput', () => {
+    it('should return correct input and loggerInfo', () => {
+      const fileName = '98498465654849896546.pdf'
+      const electionId = 'blegh'
+      const nationalIdActor = '1234564321'
+      const nationalIdClient = '1234564321'
```

Comment:
It might be a good idea to have different nationalIds here since you are checking for the each one distinctly.

Reply:
done